src/react-aria-modal.js

```diff
@@ -166,9 +167,9 @@ let Modal = class Modal extends React.Component {
       dialogProps.tabIndex = '-1';
     }
 
-    const childrenArray = [React.DOM.div(dialogProps, props.children)];
+    const childrenArray = [DOM.div(dialogProps, props.children)];
```

Comment:
> Why wasn't that used instead of React.DOM.div?

I don't recall — probably because it was there? Let's give `React.createElement` a shot to avoid the added dependency.